controllers/databaseclusterrestore_controller.go

```diff
@@ -254,6 +253,12 @@ func (r *DatabaseClusterRestoreReconciler) restorePSMDB(ctx context.Context, res
 					Region:            backupStorage.Spec.Region,
 					EndpointURL:       backupStorage.Spec.EndpointURL,
 				}
+			case everestv1alpha1.BackupStorageTypeAzure:
```

Comment:
What about the support of GCS?

Reply:
Per our discussion, psmdb does not have GCS support yet.

Reply:
The conversation is not resolved. We need to add PXC/PSMDB handles for GCS storage and make the configuration using S3 fields for the PXC/PSMDB operators because S3 in those operators stands for GCS/S3 and these operators supports that. Also, We need to have tests for that as well

Reply:
Yet, there should be a comment that explains this logic because it looks a bit misleading